### ISSUE TITLE
Lists of information should be marked up as HTML lists

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#1427]
**Acceptance Criteria:**
- [ ] Plan a lesson link section (the circle link icons to different lesson components) should be a list of links
- [ ] About us link section (who we are, leadership etc) should be a list of links
- [ ] List of board/leadership cards in about us should be lists
- [ ] Documents on the about-us/board page should be a list
- [ ] List of partner logos on about-us/partners should be a list
- [ ] List of subjects on a subject listing page

**Comment by jack-skerrett-oak** [comment]
https://www.notion.so/oaknationalacademy/The-links-within-the-Plan-a-lesson-page-are-visually-a-list-but-not-marked-up-as-one-in-HTML-164249ab85d84a788778848a7d03c7ca?pvs=4
https://www.notion.so/oaknationalacademy/The-in-page-nav-for-the-About-us-pages-is-marked-up-as-a-nav-but-the-items-are-not-in-a-list-ef5a71b2200b4e63a33c8f87032fa9b3?pvs=4
https://www.notion.so/oaknationalacademy/The-team-members-are-presented-in-a-list-but-the-HTML-structure-of-the-section-is-not-reflective-of--e40399aec3ce42d59aa0a8dc98e2f5c0?pvs=4
https://www.notion.so/oaknationalacademy/The-team-members-are-presented-in-a-list-but-the-HTML-structure-of-the-section-is-not-reflective-of--ae4334f344624e9dafe8e1ecc66a13c7?pvs=4
https://www.notion.so/oaknationalacademy/The-documents-available-on-the-page-are-presented-in-a-list-but-the-HTML-structure-is-not-reflective-a7040ef35804402f86106eca10fbb480?pvs=4
https://www.notion.so/oaknationalacademy/The-partner-logos-are-presented-in-a-list-but-the-HTML-structure-is-not-reflective-of-this-3686da85c3414b599cc6187ff2fd1707?pvs=4

**Comment by jack-skerrett-oak** [comment]
moved to notion
